platform-ui-skin/src/main/webapp/skin/less/social/skin/TopBar/Style.less

```diff
@@ -231,6 +231,8 @@ body {
               }
 
               .media {
+                display: flex;
+                align-items: center;
```

Comment:
No need to align the avatar and the body of the notification.
![image](https://user-images.githubusercontent.com/677135/127546438-0c424e2b-437d-427a-a8a6-0c226cd73523.png)
I think it is better this way to have the avatar and th eusern ame aligned
![image](https://user-images.githubusercontent.com/677135/127546570-d6c97314-4d2a-413d-a854-a66e76dedb9e.png)
 
```suggestion
                display: flex;
```